test/avrora/codec/plain_test.exs

```diff
@@ -106,6 +112,11 @@ defmodule Avrora.Codec.PlainTest do
       assert encoded == payment_message()
     end
 
+    test "when payload with null value is matching the schema and schema is usable" do
+      {:ok, encoded} = Codec.Plain.encode(null_payment_payload(), schema: payment_schema())
```

Comment:
To be consistent, can you make 1 blank line right before the assertion (like you did in the case above)